hs_core/hydroshare/resource.py

```diff
@@ -263,7 +267,7 @@ def create_resource(
         resource_type, owner, title,
         edit_users=None, view_users=None, edit_groups=None, view_groups=None,
         keywords=(), metadata=None, extra_metadata=None,
-        files=(), fed_res_file_names='', fed_res_path='', fed_copy_or_move=None,
+        files=(), source_names=[], fed_res_path='', move=False,
```

Comment:
Updated docstrings to clarify, and eliminated all use of source_names as a string. There was one remaining place where it **was** a string.